app/src/features/surveys/view/survey-animals/TelemetryMap.tsx

```diff
@@ -0,0 +1,142 @@
+import { Box, Paper, Typography } from '@mui/material';
+import MapContainer from 'components/map/MapContainer';
+import { Feature } from 'geojson';
+import L, { LatLng } from 'leaflet';
+import moment from 'moment';
+import { useMemo, useState } from 'react';
+import { GeoJSON } from 'react-leaflet';
+import { uuidToColor } from 'utils/Utils';
+import { v4 } from 'uuid';
+import { IAnimalDeployment, ITelemetryPointCollection } from './device';
+
+interface ITelemetryMapProps {
+  telemetryData?: ITelemetryPointCollection;
+  deploymentData?: IAnimalDeployment[];
+}
+
+type ColourDeployment = IAnimalDeployment & { colour: string; fillColour: string };
+
+interface ILegend {
+  hasData: boolean;
+  colourMap: ColourDeployment[];
+}
+
+const Legend = ({ hasData, colourMap }: ILegend) => {
+  return (
+    <div className="leaflet-bottom leaflet-left">
+      <div className="leaflet-control leaflet-bar">
+        <Paper sx={{ padding: 2 }}>
+          {hasData ? (
+            colourMap.map((deploymentAndColour, idx) => (
+              <Box
+                key={deploymentAndColour.deployment_id}
+                display={'flex'}
+                flexDirection={'row'}
+                alignItems={'center'}
+                marginTop={idx > 0 ? 1 : 0} /*<- Any better way to do this? */
```

Comment:
There is. Give `sx={{ gap: 1 }}` to the parent element, and it will put a gap between all the child elements (note this is a Flexbox property)